pkgs/development/libraries/science/math/ecos/default.nix

```
@@ -29,9 +29,10 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "A lightweight conic solver for second-order cone programming";
-    homepage = https://www.embotech.com/ECOS;
+    homepage = "https://www.embotech.com/ECOS";
+    downloadPage = "https://github.com/embotech/ecos";
```

I'll defer to someone like @jonringer, but is `downloadPage` an attribute we commonly put in `meta`?  Seems like it can be easily found from just looking at where `src` is downloading from.

---

commonly? no, but it is a support attribute, however, for github, you usually link to the releases page as that's where you would go to download a given version
```suggestion
    downloadPage = "https://github.com/embotech/ecos/releases";
```

---

I thought it was somewhat useful as an easy link for reviewers/committers for getting the project, though it doesn't really contain more information than ``fetchFromGitHub``. Resolving @jonringer 's comment.